src/main/java/org/medibloc/panacea/domain/DIDs.java

```diff
@@ -24,11 +24,9 @@ private static String encodePubKey(byte[] pubKey) throws NoSuchAlgorithmExceptio
     }
 
 
-    public static VerificationMethod createVerificationMethod(DIDWallet wallet, String key) throws NoSuchAlgorithmException {
+    public static VerificationMethod createVerificationMethod(String did, DIDWallet wallet, String key) {
```

Comment:
I changed the did value to be taken from outside.
Because verificationMethodId is similar to did.
If you create multiple verificationMoethods, verificationMethodId is affected to did.

Reply:
I think priv/pub key can always be different, verificationMethodId is similar to did.

Reply:
Yes. That's true :+1: